indexer/docker-compose.yml

```diff
@@ -75,5 +75,121 @@ services:
       postgres:
         condition: service_healthy
 
+  gateway-frontend:
```

Comment:
Since the gateway isn't open source yet, it is appropriate it add it to the compose file? I'd imagine this docker configuration would exist in the gateway repo vs the indexer

Reply:
The docker images are published publicly. THe docker configuration needs to exist in both ideally for our own convenience. We could consider removing it once everything is running but in meantime seperating them needlessly slows us down